src/Time/_TimeItem.test.js

```diff
@@ -0,0 +1,176 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import Time from './Time';
+
+describe('Time Item Prop spreading', () => {
```

Comment:
Should just reference `TimeItem` as the top-level `describe`.  Since there are multiple prop spreading tests, you could put another prop spreading `describe` and then move all those tests into that.  Just better test organization.